test/functional/includeconf.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
I'm not sure a neat and tidy way to test that. Should I just make multiple classes, one for each, and then call them one at a time in the bottom `if __name__ == '__main__':`?